nevergrad/functions/multiobjective/core.py

```diff
@@ -74,8 +75,7 @@ def __call__(self, *args: Any, **kwargs: Any) -> float:
         # The following is not. It should be called locally.
         return self.compute_aggregate_loss(losses, *args, **kwargs)
 
-    @property
-    def pareto_front(self) -> List[ArgsKwargs]:
+    def filter_pareto_front(self):
```

Comment:
You should probably make this private: `_filter_pareto_front`

Reply:
Done.